tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -378,3 +378,43 @@ DRAW_TEST(CGContext, PremultipliedAlphaImage) {
     CGContextSetRGBFillColor(context, 1.0, 0.0, 0.0, 0.5);
     CGContextFillRect(context, { 0, 0, 100, 100 });
 }
+
+DRAW_TEST_F(CGContext, AntialiasToggleTranscendsGState, WhiteBackgroundTest<>) {
```

Comment:
@aballway it is impossible to query the antialias state of the context, and this allowed us to check against the reference platform